chaco/plot.py

```diff
@@ -17,34 +17,34 @@
 from .array_data_source import ArrayDataSource
 from .array_plot_data import ArrayPlotData
 from .base_xy_plot import BaseXYPlot
-from .barplot import BarPlot
-from .candle_plot import CandlePlot
-from .colormapped_scatterplot import ColormappedScatterPlot
-from .contour_line_plot import ContourLinePlot
-from .contour_poly_plot import ContourPolyPlot
-from .cmap_image_plot import CMapImagePlot
 from .data_range_1d import DataRange1D
 from .data_view import DataView
 from .default_colormaps import Spectral
 from .grid_data_source import GridDataSource
 from .grid_mapper import GridMapper
 from .image_data import ImageData
-from .image_plot import ImagePlot
 from .legend import Legend
-from .lineplot import LinePlot
-from .line_scatterplot_1d import LineScatterPlot1D
 from .linear_mapper import LinearMapper
 from .log_mapper import LogMapper
+from .plots.barplot import BarPlot
+from .plots.candle_plot import CandlePlot
+from .plots.colormapped_scatterplot import ColormappedScatterPlot
+from .plots.contour.contour_line_plot import ContourLinePlot
+from .plots.contour.contour_poly_plot import ContourPolyPlot
+from .plots.cmap_image_plot import CMapImagePlot
+from .plots.filled_line_plot import FilledLinePlot
+from .plots.image_plot import ImagePlot
+from .plots.jitterplot import JitterPlot
 from .plot_label import PlotLabel
```

Comment:
nit-pick : this `plot_label` import is out of order 😅 
`plot_label` would come before `plots.*` imports I think